tools/toolchains/gcc.py

```diff
@@ -33,7 +33,7 @@ class GCC(mbedToolchain):
     STD_LIB_NAME = "lib%s.a"
     DIAGNOSTIC_PATTERN = re.compile('((?P<file>[^:]+):(?P<line>\d+):)(?P<col>\d+):? (?P<severity>warning|[eE]rror|fatal error): (?P<message>.+)')
 
-    GCC_RANGE = (LooseVersion("6.0.0"), LooseVersion("7.0.0"))
+    GCC_RANGE = (LooseVersion("8.0.0"), LooseVersion("9.0.0"))
```

Comment:
This looks like it will now reject gcc version 6.x and 7.x, forcing users to upgrade gcc. Is that what is intended?

Reply:
It's not really "forced". As at present, the build system will proceed if GCC is outside that range - you just get the warning message to indicate that this isn't the version we're testing ourselves and hence officially supporting.

(I've been using the unsupported GCC 7 for ages locally, getting the warning that it's not GCC 6 every time I build, with no issues).

I don't think there's any reason why GCC 6 or 7 would actually stop working in this release, although no doubt eventually something will be incompatible in some future release once they're not tested any more.

Reply:
Ok, thought that might be the case, but worth checking. We're taking on the tools here in CPC1 and are just starting to get familiar with them. Thanks for replying.